.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,7 @@
   "name": "Supervisor dev",
   "image": "ghcr.io/home-assistant/devcontainer:supervisor",
   "appPort": ["9123:8123", "7357:4357"],
-  "postCreateCommand": "bash devcontainer_bootstrap",
+  "postCreateCommand": "sudo bash devcontainer_bootstrap",
```

Comment:
I think we should do this instead <https://github.com/home-assistant/supervisor/pull/3967>

Reply:
Then adjust the user groups, don't just slap on a "sudo".
Also set the user directly, as that is what has changed now, so that does not give more issues in the future.

Reply:
Honestly replacing `sudo` with `sudo -u root -g root` sounds like a big non-op to me.

Reply:
> Honestly replacing `sudo` with `sudo -u root -g root` sounds like a big no-op to me.

Right, I interpreted @ludeeus message wrong above. I reverted that change.

Instead, this change now relies on [#32](https://github.com/home-assistant/devcontainer/pull/32), which only selectively uses `sudo`. The current implementation creates `test_supervisor` still root owned (as the actual Supervisor will anyways create things as root). But since that is not part of the git repository, its not problematic.